connector/signaltometricsconnector/README.md

```diff
@@ -225,3 +225,12 @@ signaltometrics.service.name: <service_name_of_the_otel_collector>
 signaltometrics.service.namespace: <service_namespace_of_the_otel_collector>
 signaltometrics.service.instance.id: <service_instance_id_of_the_otel_collector>
 ```
+
+### Custom OTTL functions
+
+The component implements a couple of custom OTTL functions:
```

Comment:
Is this approach validated in general? I'm worried that if we proceed with this we open the door for components to start implementing custom ottl functions. Is this validated already or happening anywhere else in the project?

Isn't possible to introduce them in `ottl` package at first place?

Reply:
> Is this approach validated in general?

Not sure if it is accepted but it is used. For example: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/transformprocessor/README.md#supported-functions has a bunch of extra OTTL functions that it adds.

> Isn't possible to introduce them in ottl package at first place?

It is and that is the plan but I didn't want to block the component. Here are the 2 issues to resolve the 2 custom OTTL functions introduced in this PR:

https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/35621
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/36572

Reply:
Makes sense! I also validated this approach with the maintainers and we should be good to go that way.